code/__defines/mobs.dm

```diff
@@ -84,7 +84,7 @@
 #define APPEARANCE_EYE_COLOR 0x100
 #define APPEARANCE_ALL_HAIR (APPEARANCE_HAIR|APPEARANCE_HAIR_COLOR|APPEARANCE_FACIAL_HAIR|APPEARANCE_FACIAL_HAIR_COLOR)
 #define APPEARANCE_ALL       0xFFFF
-#define APPEARANCE_PLASTICSURGERY (APPEARANCE_HAIR|APPEARANCE_HAIR_COLOR|APPEARANCE_FACIAL_HAIR|APPEARANCE_FACIAL_HAIR_COLOR|APPEARANCE_GENDER) & ~APPEARANCE_RACE
+#define APPEARANCE_PLASTICSURGERY (APPEARANCE_HAIR|APPEARANCE_HAIR_COLOR|APPEARANCE_FACIAL_HAIR|APPEARANCE_FACIAL_HAIR_COLOR|APPEARANCE_GENDER|APPEARANCE_SKIN|APPEARANCE_EYE_COLOR) & ~APPEARANCE_RACE
```

Comment:
`#define APPEARANCE_PLASTICSURGERY (APPEARANCE_ALL & ~APPEARANCE_RACE)`? Much shorter, and reinforces the fact that it's everything but species.